### DISCUSSION TITLE
Contribution guide for Beginners.

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [ammargitham/WallFlow#20] [Q&A] [answered]
I want to contribute to the project. Currently i only know c++ and haven't contributed anywhere yet or done any actual development.

If you can please tell me what would i need to study to be able to contribute in this project. Any guidance?

**Comment by ammargitham** [comment, marked as answer]
Hey! You will have to start with Kotlin first. And then dive into Android.

Some getting started guides:

Kotlin: https://kotlinlang.org/docs/getting-started.html
Android: https://developer.android.com/get-started/overview

**Reply by KAGEYAM4** [reply]
Thank you.
